pkgs/development/python-modules/thorlabspm100/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, unittestCheckHook
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "thorlabspm100";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "clade";
+    repo = "ThorlabsPM100";
+    rev = version;
```

```suggestion
    rev = "v${version}";
```
or `rev = "refs/tags/v${version}"`